core-bundle/contao/themes/flexible/styles/basic.css

```diff
@@ -69,6 +69,9 @@
 	--serp-preview-title: #1a0dab;
 	--nested-bg: #fbfbfd;
 	--search-field-bg: rgba(255, 255, 255, .2);
+	--choices-highlighted: #3997fe;
+	--choices-bg: #eee;
+	--choices-border: #bbb;
```

Comment:
I would prefer to have this in the .pcss file so there are no potential remnants if we remove it in the future. WDYT?

Reply:
Or should we maybe use generic names in the `basic.css` and reference/rename them in the `.pcss` file? So that all color definitions are still at one place.

Reply:
See [f810e21](https://github.com/contao/contao/pull/8261/commits/f810e21b238d8079cc7abf36f2d2d864d0bad0e1)